aten/src/ATen/native/TensorShape.cpp

```diff
@@ -1055,6 +1055,12 @@ std::vector<Tensor> unbind(const Tensor &self, int64_t dim) {
   return tensors;
 }
 
+#ifdef BUILD_NAMEDTENSOR
+std::vector<Tensor> unbind(const Tensor& self, Dimname dim) {
+  return at::unbind(self, dimname_to_position(self, dim));
```

Comment:
I'd expect to see some change in the existing unbind method to remove the one name (if named) from each resulting unbound tensor. Is that not the expected behavior?

Reply:
`unbind` calls `select` which does that already.